src/libraries/System.Net.Http/src/System/Net/Http/Headers/KnownHeaders.cs

```diff
@@ -221,6 +226,13 @@ public BytePtrAccessor(byte* p, int length)
                     }
                     break;
 
+                case 9:
+                    switch (key[0])
```

Comment:
Unrelated, but would this routine get better codegen if these inner switches were switching on `key[0] | 0x20` instead of having two cases, one for the upper-, and other for the lower-case?